wiki/en/Software-Manual.md

```diff
@@ -119,14 +117,70 @@ You can group users together using the "group" toggle. Moving the fader of any m
 
 ### Mute button
 
-Prevents users being heard in your local mix. Be aware that when you mute someone, they will see a "muted" icon above your fader to indicate that you cannot hear them. Note also that you will continue to see their VU meters moving if sound from the muted user is reaching the server. Your fader position for them is also unaffected. 
+Prevents users being heard in your local mix. Be aware that when you mute someone, they will see a "muted" icon above your fader to indicate that you cannot hear them. Note also that you will continue to see their VU meters moving if sound from the muted user is reaching the server. Your fader position for them is also unaffected.
 
 Note that muting your **own** channel only means you will not hear your signal from the server (and is not advised as it can lead to you becoming out of time with other players). This is therefore not the same as using "[Mute Myself](#mute-myself-button)".
 
 ### Solo button
 
 Allows you to hear one or more users on their own. Those not soloed will be muted. Note also that those people who are not soloed will see a "muted" icon above your fader.
 
+## Menu commands
+
+### File > Connection Setup...
+
+Opens the [connection dialogue (see above)](#connectdisconnect-button).
```

Comment:
Are we consistent with dialogue vs dialog?

Reply:
Between the two places, yes.  I didn't check the whole of the wiki.  (Personally, if we're talking about UI widgets like "dialog", I think we should use the technical term rather than a British English word meaning communication between people.)